torch/nn/parallel/comm.py

```diff
@@ -235,3 +242,8 @@ def gather(tensors, dim=0, destination=None, *, out=None):
                 "'destination' must not be specified when 'out' is specified, but "
                 "got destination={}".format(destination))
         return torch._C._gather_out(tensors, out, dim)
+
+
+def _handle_complex(tensor):
+    return torch.view_as_real(tensor) if not isinstance(tensor,
+                                                        UninitializedParameter) and tensor.is_complex() else tensor
```

Comment:
Curious, why do we need to check if it's UninitializedParameter?

Reply:
Can't call is_complex if it's uninitialized, I'll add a comment